engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -6,6 +6,8 @@ import java.util.{Date, UUID}
 import javax.sql.rowset.serial.SerialClob
 
 import _root_.slick.backend.DatabaseConfig
+import _root_.slick.dbio
+import _root_.slick.dbio.Effect.Read
 import _root_.slick.driver.JdbcProfile
```

Comment:
http://stackoverflow.com/questions/687071/what-is-the-root-package-in-scala

That said, when I've had intellij add it for me in the past, it's always been unnecessary and due to some operator error combined w/ intellij